ocis-pkg/config/configlog/log.go

```diff
@@ -0,0 +1,30 @@
+package configlog
+
+import (
+	"fmt"
+	"os"
+)
+
+// LogError logs the error
+func LogError(err error) {
```

Comment:
Great. But they stutter as they are callled like `configlog.LogError` maybe just call them `Error` `ReturnErrror` `ReturnFatal`? So then they would be used like `configlog.Error`

Obviously not critical as some things just stutter....